lib/gooddata/bricks/pipeline_base.rb

```diff
@@ -0,0 +1,26 @@
+# encoding: UTF-8
+# frozen_string_literal: true
+#
+# Copyright (c) 2010-2022 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+module GoodData
+  module Bricks
+    class PipelineBase
```

Comment:
Change to BasePipeline for consitence, change file name also please.

Reply:
Done update code. Thank you.